tests/benchmarks/makefile

```diff
@@ -18,7 +18,7 @@ CONFIG_MK = $(MFEM_BUILD_DIR)/config/config.mk
 MFEM_LIB_FILE = mfem_is_not_built
 -include $(CONFIG_MK)
 
-SEQ_TESTS = bench_ceed bench_tmop bench_vector bench_virtuals
+SEQ_TESTS = bench_ceed bench_tmop bench_vector bench_virtuals bench_assembly
```

Comment:
Can the alphabetical ordering be kept?

Reply:
done in [7b70e27](https://github.com/mfem/mfem/pull/2947/commits/7b70e27f1fc14f6f06c2231699205c6df74533af)